Hadoop/manual.md

```diff
@@ -0,0 +1,63 @@
+Минимальные технические требования:
+- RAM: > 2 Gb;
+- CPUs: > 1 2.1MHz;
+- Разрядность - 64-bit version.
+- Свободного дисковое пространство > 15 Gb.
+Программные требования:
+- Docker-compose (2 and later);
+- Docker;
+
+!!! ALL COMMANDS must be executed from context where docker-compose files are situated !!!
+
+
+## To build images locally and start, run theese commands
+
+```
+docker-compose -f docker-compose_build.yaml && docker-compose -f docker-compose_build.yaml up -d
+```
+
+### to stop them
+
+```
+docker-compose -f docker-compose_build.yaml down
+```
+
+## To download images and start containers, run this command
+
+```
+docker-compose up -d
```

Comment:
Shal we add instructions on how to make it reachable outside of the virtual machine?
![image](https://user-images.githubusercontent.com/43520795/154980109-e31e451a-05ec-44c0-abf3-4ac65295e5c2.png)

Reply:
becouse of security, it is better to make it reachable via proxy (exmpl, nginx)

Reply:
Ok, but we have a "dumb" user who doesn't know how to do that. Can we do something to make it work "out of the box"?

Reply:
just to change docker-compose.yml

Reply:
Seems legit)
Approved